filebeat/module/tidb/slowlog/_meta/fields.yml

```diff
@@ -0,0 +1,281 @@
+- name: slowlog
+  type: group
+  description: >
+    Contains fields from the TiDB slow logs.
+  fields:
+    - name: Time
+      type: date_nanos
+      description: >
+        The print time of log.
```

Comment:
Is it possible to not having this hard coded fields? As these fields are likely to change between TiDB versions.

Reply:
It's not possible to ignore fields. 
Filebeat module requires that each field's type and description be specified in the spec, and will check in CI test whether the output matches the spec, and report an error if it does not.

In addition, the automatically generated template, index mapping, and documentation, all rely on field spec.

So I think it must be hard coded.

Reply:
Got it. How about we document the TiDB versions some where, so that it can be easy for the follow maintainers to know it is outdated?

Reply:
I'll document TiDB version in module source root. I haven't found a good automated update method yet.